Classes/New Issue/GithubClient+Template.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  GithubClient+Template.swift
+//  Freetime
+//
+//  Created by Ehud Adler on 11/11/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import GitHubAPI
+import GitHubSession
+import Squawk
+
+private let githubIssueURL = ".github/ISSUE_TEMPLATE"
+
+
+struct IssueTemplateRepoDetails {
+    let repo: RepositoryDetails
+    let defaultBranch: String
+}
+
+struct IssueTemplateDetails {
+    let repoDetails: IssueTemplateRepoDetails
+    let session: GitHubUserSession?
+    let viewController: UIViewController
+    weak var delegate: NewIssueTableViewControllerDelegate?
```

Comment:
@rnystrom We can't reduce the networking calls because the title and description of files are at the top of the files. The file names are not equal to the template names. Let me know if that makes sense. I will make the changes regarding passing the templates to the VC and letting it take care of displaying the alert.